warehouse/query-core/src/main/java/datawave/query/tracking/ActiveQueryLog.java

```diff
@@ -35,7 +35,7 @@
     
     private Cache<String,ActiveQuery> CACHE = null;
     private ReentrantReadWriteLock cacheLock = new ReentrantReadWriteLock();
-    private Timer timer = new Timer("ActiveQueryLog");
+    private Timer timer = null;
```

Comment:
timer needs to start as null so that we can differentiate between when it is first being initialized and when it is being reinitialized